source/faq.rst

```diff
@@ -1,264 +1,219 @@
 
-MySQL metrics are collected with different resolutions (5 seconds, 10 seconds,
-and 60 seconds by default). Linux and MongoDB metrics are collected with 1
-second resolution.
 
-In case of bad network connectivity between |pmm-server| and |pmm-client| or
-between |pmm-client| and the database server it is monitoring, scraping every
-second may not be possible when latency is higher than 1 second.
-
-You can change the minimum resolution for metrics by the following way:
+.. _metrics-resolution:
 
-#. Select the |pmm-settings| dashboard in the main menu.
+************************************
+What resolution is used for metrics?
+************************************
 
-   .. figure:: .res/graphics/png/pmm-add-instance.png
+The default values are:
 
-      Choosing the |pmm| *Settings* menu entry
+- Low: 60 seconds
+- Medium: 10 seconds
+- High: 5 seconds
 
-#. In the *Settings* section, choose proper metrics resolution with the slider.
-   The tooltip of the slider will show you actual resolution values.
+(See :ref:`server-admin-gui-metrics-resolution`.)
 
-   .. figure:: .res/graphics/png/pmm.settings_settings.png
 
-      Choosing metrics resolution on the *Settings dashboard*
 
-#. Click the *Apply changes* button.
+.. _alertmanager:
+.. _how-to-setup-alerting-with-grafana:
 
-.. note:: Consider increasing minimum resolution
-   when |pmm-server| and |pmm-client| are on different networks,
-   or when :ref:`pmm.amazon-rds`.
+********************************
+How do I set up Alerting in PMM?
+********************************
 
-.. _alertmanager:
+When a monitored service metric reaches a defined threshold, PMM Server can trigger alerts for it either using the Grafana Alerting feature or by using an external Alertmanager, a high-performance solution developed by the Prometheus project to handle alerts sent by Prometheus.
+(Both are advanced features that require knowledge of third-party tools.)
 
-How to set up Alerting in PMM?
-================================================================================
+With these methods you must configure alerting rules that define conditions under which an alert should be triggered, and the channel used to send the alert (e.g. email).
 
-You can make PMM Server trigger alerts when your monitored service reaches some thresholds in two ways:
+Grafana Alerts are already integrated into PMM Server and may be simpler to get set up.
 
-* using `Grafana Alerting feature <https://grafana.com/docs/grafana/latest/alerting/rules/>`_,
-* using external `Alertmanager <https://github.com/prometheus/alertmanager>`_ (a
-  high-performance solution developed by the Prometheus project to handle alerts sent
-  by Prometheus).
+Alertmanager allows the creation of more sophisticated alerting rules and can be easier to manage installations with a large number of hosts. This additional flexibility comes at the expense of simplicity and requires advanced knowledge of Alertmanager rules.
```

Comment:
Maybe a quibble but why do we keep refering to "advanced knowledge". I think that's initimidating language, and just knowing how to write alerting rules to me doesn't seem advanced. I'd drop it in order to make the conversation more approachable to new users.

Reply:
Not a quibble, you're right. I guess the original intent was as a gentle warning.